### ISSUE TITLE
A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_ColorContrast : The image of text "v0.23.0" and "550.21K" has color contrast ratio less than 4.5:1.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-azurefunctions#2285]
**“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**

GitHub Tags:#A11y_Azure Extensions for VS Code;#Azure Tools for VS Code;#A11yTCS;#A11yMAS;#MAC;#FTP;#Benchmark;#ColorContrast;#A11ySev2:#MAS1.4.3;

### Environment Details:
Application Name: Azure Extensions for VS Code
Application Version: 1.48.0-insider
Commit: cfbd1999769f4f08dce29629fb92fdc0fac53829
Date: 2020-08-06T05:40:40.871Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina Version 10.15.6 (19G73)
Screen Reader: VoiceOver
MAS References: MAS1.4.3

### Repro Steps:
1. Open Visual studio code
2. Click on extension button and search for "Azure Functions"
3. Click on install and move to the "Azure" button, the "Default Screen" will appear.
4. Navigate to image of text "v0.23.0" and "550.21K" and verify if the color contrast ratio is less than 4.5:1 or not.

### Actual:
The image of text "v0.23.0" and "550.21K" has color contrast ratio of 2.2:1 which is less than the standard color contrast ratio of 4.5:1 which is not appropriate as the users with low vision will have difficulty in reading the text on the screen.

### Expected:
The color contrast ratio of image of text "v0.23.0" and "550.21K" should be greater than or equal to 4.5:1.

### User Impact:
If the color contrast ratio of the content present on the screen is less than 4.5:1 then the users with low vision will have difficulty in reading the data on the screen.

### Attachment link for Reference:
<img width="1379" alt="2285_A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_ColorContrast The image of text  v0 23 0  and  550 21K  has color contrast ratio less than 4 51" src="https://user-images.githubusercontent.com/52036772/89658052-ad063200-d8eb-11ea-93c3-3db82efdf328.png">

**Comment by EricJizbaMSFT** [comment]
Those are not necessary, so I just removed them.

<img width="693" alt="Screen Shot 2020-08-25 at 3 27 51 PM" src="https://user-images.githubusercontent.com/11282622/91234167-8f0c4000-e6e7-11ea-9f50-d81eebc1112a.png">

**Comment by MayuriNaik** [comment]
Verified the bug in Visual Studio Code - Insiders application (Version: 1.50.0-insider) and Azure Functions (v0.24.0). The controls such as "Visual studio marketplace v0.23.0", "installs 550.21k" and "Azure Pipelines failed" are now no more available. Hence closing the bug. PFA
<img width="1116" alt="2285_Closed" src="https://user-images.githubusercontent.com/52036772/93176569-14b36680-f74f-11ea-86cb-49c80616ca2d.png">
